third_party/gpus/nvidia_common_rules.bzl

```diff
@@ -331,10 +332,13 @@ def _download_redistribution(
         strip_prefix = repository_ctx.attr.override_strip_prefix
     else:
         strip_prefix = archive_name
-    repository_ctx.extract(
-        archive = file_name,
-        stripPrefix = strip_prefix,
-    )
+    if url.endswith(".tar.xz") or url.endswith(".tar"):
+        extract_tar_with_tar_tool(repository_ctx, file_name, strip_prefix)
+    else:
+        repository_ctx.extract(
```

Comment:
Notice, here you call extract explicitly (which is fine) and in the other file you have that unnecessary _extract_tar_with_bazel method

Reply:
Ack.